### ISSUE TITLE
Compiling sphinx on Windows w/ rtd_theme gives red class names

### ISSUE THREAD

**Issue by MatthiasKauer** [snide/sphinx_rtd_theme#230]
Hi,
I'm in the process of setting up documentation with sphinx. We chose this theme and began. I have now checked out the docs and they look significantly different on Windows. I have created the following example.
Conf.py has only autodoc extension and is otherwise standard. sphinx 1.3.1, Python 3 via anaconda.

```
#mytest.py
def test():
    """
    prints hello, that's it
    """
    print("hallo")

class TestClass(object):
    """the test class"""
    def __init__(self):
        self.var = 0
```

```
#index.rst
test module
=========================

.. automodule:: mytest
    :members:
```

On Windows, the output after `make html` has all links in red. In Linux everything appears to work normally (I haven't tested this MWE there, I'm extrapolating from my larger project).
![sphinx-rtd-windows](https://cloud.githubusercontent.com/assets/2436121/8978589/0acc0ea4-36d6-11e5-8afe-b0360c25df55.jpg)

Any ideas where this comes from?

Kind regards,
Matthias

**Comment by agjohnson** [comment]
What version of this theme are you using?

**Comment by MatthiasKauer** [comment]
I'm not sure if this is the best way to test but using conda, I get the following.
```
[py34] C:\dvl\transferdyn\docs>conda install sphinx_rtd_theme
....
# All requested packages already installed.
....
sphinx_rtd_theme          0.1.7                    py34_0
```

**Comment by agjohnson** [comment]
This was resolved in 0.1.8, by #191

**Comment by Blendify** [comment]
@ericholscher please close this issue.
